Assets/MixedRealityToolkit/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "com.microsoft.mixedrealitytoolkit",
```

Comment:
>com.microsoft.mixedrealitytoolkit [](start = 10, length = 33)

If we are going with this name pattern
, we should name this "com.microsoft.mixedreality.toolkit"

Reply:
I'll change this and update the PR when I get a chance

Reply:
Package.json files updated